js/core/utils/size.js

```diff
@@ -140,24 +144,55 @@ export const getVisibleHeight = function(element) {
-export const getInnerWidth = (el) => elementSize(el, 'innerWidth');
-export const setInnerWidth = (el, value) => elementSize(el, 'innerWidth', value);
-export const getInnerHeight = (el) => elementSize(el, 'innerHeight');
-export const setInnerHeight = (el, value) => elementSize(el, 'innerHeight', value);
+export const commonCallbacks = {
+    getWidth: function(el, value) { return elementSizeHelper('width', ...arguments); },
+    setWidth: function(el, value) { return elementSizeHelper('width', ...arguments); },
+    getHeight: function(el, value) { return elementSizeHelper('height', ...arguments); },
+    setHeight: function(el, value) { return elementSizeHelper('height', ...arguments); },
+    getOuterWidth: function(el, value) { return elementSizeHelper('outerWidth', ...arguments); },
+    setOuterWidth: function(el, value) { return elementSizeHelper('outerWidth', ...arguments); },
+    getOuterHeight: function(el, value) { return elementSizeHelper('outerHeight', ...arguments); },
+    setOuterHeight: function(el, value) { return elementSizeHelper('outerHeight', ...arguments); },
+    getInnerWidth: function(el, value) { return elementSizeHelper('innerWidth', ...arguments); },
+    setInnerWidth: function(el, value) { return elementSizeHelper('innerWidth', ...arguments); },
+    getInnerHeight: function(el, value) { return elementSizeHelper('innerHeight', ...arguments); },
+    setInnerHeight: function(el, value) { return elementSizeHelper('innerHeight', ...arguments); },
+};
+function elementSizeHelper(sizeProperty, el, value) {
+    return arguments.length === 2 ? elementSize(el, sizeProperty) : elementSize(el, sizeProperty, value);
+}
+
+export const getWidth = function(el, value) { return commonCallbacks.getWidth(...arguments); };
+export const setWidth = function(el, value) { return commonCallbacks.setWidth(...arguments); };
+export const getHeight = function(el, value) { return commonCallbacks.getHeight(...arguments); };
+export const setHeight = function(el, value) { return commonCallbacks.setHeight(...arguments); };
+export const getOuterWidth = function(el, value) { return commonCallbacks.getOuterWidth(...arguments); };
+export const setOuterWidth = function(el, value) { return commonCallbacks.setOuterWidth(...arguments); };
+export const getOuterHeight = function(el, value) { return commonCallbacks.getOuterHeight(...arguments); };
+export const setOuterHeight = function(el, value) { return commonCallbacks.setOuterHeight(...arguments); };
+export const getInnerWidth = function(el, value) { return commonCallbacks.getInnerWidth(...arguments); };
+export const setInnerWidth = function(el, value) { return commonCallbacks.setInnerWidth(...arguments); };
+export const getInnerHeight = function(el, value) { return commonCallbacks.getInnerHeight(...arguments); };
+export const setInnerHeight = function(el, value) { return commonCallbacks.setInnerHeight(...arguments); };
```

Comment:
It looks like `commonCallbacks` usage breaks Tree Shaking for these methods.

Reply:
All of these methods share the same implementation (the `elementSize` function). So the `commonCallbacks` costs us only 10 LOCs